docs/source/tt-metalium/tt_metal/environment_variables/index.rst

```diff
@@ -0,0 +1,75 @@
+.. _Environment Variables:
+
+Environment Variables
+=====================
+
+TT_METAL_HOME
+-------------
+
+**Required:** Conditional (only when required by a given workflow)
+
+**Description:**
+
+``TT_METAL_HOME`` is an environment variable that points to the root directory of the tt-metal repository. It may be needed before running some tests, or scripts. It is not mandated by the metal runtime.
+
+**Usage:**
+
+.. code-block:: bash
+
+   export TT_METAL_HOME=/path/to/your/tt-metal
+
+**When is it needed:**
+
+- **Development:** Used by test scripts and build tools to locate repository files
+- **Documentation builds:** Referenced when checking spelling and building docs
+- **CMake builds:** Used as a fallback when building tt-train standalone
+
+TT_METAL_RUNTIME_ROOT
+---------------------
+
+**Required:** Conditional (only for specific C++ application scenarios)
+
+**Description:**
+
+``TT_METAL_RUNTIME_ROOT`` serves as an override to tell ``libtt_metal.so`` where to look for its runtime artifacts (files that get read from disk).
+
```

Comment:
What does it mean for the users? I assume this is the path for the firmware artifacts? Is so

```suggestion
**Description:**

``TT_METAL_RUNTIME_ROOT`` serves as an override to tell ``libtt_metal.so`` where to look artifacts - firmware blobs, etc.. that is needed during runtime.

```